### ISSUE TITLE
Snopt 7.4 thinks an unbounded LP is bounded

### ISSUE THREAD

**Issue by hongkai-dai** [snopt/snopt-interface#24]
I tried to solve a simple LP
```
min -x(0) - x(1)
s.t 2x(0) + x(1) >= 4
    x(0) >= 0
    x(1) >= 2
```
This problem should be unbounded, but when I use the C interface of snopt 7.4-1.4, it thinks the solution (0, 4) is optimal.

The test code and snopt output log is in 
[unbounded_lp.zip](https://github.com/snopt/snopt-interface/files/2390595/unbounded_lp.zip)

When I use Snopt 7.4-1.2 with f2c interface, the solver detects unboundedness.

Is there something wrong when I set up the problem in my `problem.c` file?

**Comment by gnowzil** [comment]
Which version of the C interface are you using?  And if it's not the latest, can you please use the latest?  The problem was unbounded when I coded it up in Fortran and C.  I can't figure out which version of the C interface has `solveC` like in your example so I can't run yours directly.

**Comment by hongkai-dai** [comment]
Thanks Elizabeth! 

I used the C interface in the snopt 7.4-1.2 tar.gz file, I put the file `problem.c` in `interfaces/cexamples`.

Sorry I can't use the newest interface (like the one in snopt 7.6). We have some other problems that would fail in 7.6, as mentioned in #19 , so I testing if 7.4 would work with all our problems.

Is it OK if you could send me your C code?

**Comment by gnowzil** [comment]
https://gist.github.com/gnowzil/b1bf2f4d838f50b018fc9c03799cfa48

Same as yours, except using current interface and `m = 2` (I don't think setting `m=3` as in your example makes a difference).

**Comment by gnowzil** [comment]
You might be able to use v2.0.0.

**Comment by gnowzil** [comment]
I was able to pinpoint this bug to a pretty specific time frame (Dec 2014 to Feb 2015), which corresponds to snopt 7.4-1.1 to snopt 7.4-1.2.  The bug was fixed in 7.4-1.3.  I'm going to close this now, but feel free to let me know if there are any additional issues.

**Comment by hongkai-dai** [comment]
Thanks for looking into it!
